apps/spruce/src/pages/waterfall/styles.ts

```diff
@@ -38,3 +49,36 @@ export const InactiveVersion = styled.div`
   width: ${INACTIVE_WIDTH}px;
   text-align: center;
 `;
+
+export const statusColorMap: Record<string, string> = {
```

Comment:
super nit: statusColorMap and statusIconMap aren't actually styles. Maybe they should go in icons.ts?